src/main/views/webContentEvents.ts

```diff
@@ -298,17 +298,14 @@ export class WebContentsEventManager {
 
     private generateHandleConsoleMessage = (webContentsId: number) => (_: Event, level: number, message: string, line: number, sourceId: string) => {
         const wcLog = this.log(webContentsId).withPrefix('renderer');
-        let logFn = wcLog.verbose;
+        let logFn = wcLog.debug;
         switch (level) {
         case ConsoleMessageLevel.Error:
             logFn = wcLog.error;
             break;
         case ConsoleMessageLevel.Warning:
             logFn = wcLog.warn;
             break;
-        case ConsoleMessageLevel.Info:
-            logFn = wcLog.info;
-            break;
```

Comment:
Shouldn't we just use `console.debug` when we want debug logs?

Reply:
If the front-end exclusively did it then yes, but we want to reduce the noise that the front-end is generating, and that includes general `console.log` statements which are interpreted as `info` level.